Gemfile

```diff
@@ -20,6 +20,8 @@ gem 'twilio-ruby'
 gem 'rubocop'
 gem 'simplecov'
 
+gem 'normalize-rails' # resets base browser styles. see this url for the styles added. https://goo.gl/SzZnY2
```

Comment:
Can we put the comment on the line above? Also, this comment is not helpful at all. I have absolutely no idea what a "base browser style" is, nor why I would reset it. If nothing else, at least tag that it's related to front-end stylesheets.